### ISSUE TITLE
Triggers Naming/FileName cop with RuboCop 0.50.0

### ISSUE THREAD

**Issue by tagliala** [sds/slim-lint#65]
```
app/views/admin/users/index.html.slim:1 [W] RuboCop: Naming/FileName: The name of this source file (`index.html.slim.slim_lint.tmp20170917-92660-1io5a15.rb`) should use snake_case.
```

Workaround:

```yml
linters:
  RuboCop:
    enabled: true
    # These cops are incredibly noisy since the Ruby we extract from Slim
    # templates isn't well-formatted, so we ignore them.
    # WARNING: If you define this list in your own .slim-lint.yml file, you'll
    # be overriding the list defined here.
    ignored_cops:
      - Layout/AlignArray
      - Layout/AlignHash
      - Layout/AlignParameters
      - Layout/FirstParameterIndentation
      - Layout/IndentArray
      - Layout/IndentationConsistency
      - Layout/IndentationWidth
      - Layout/MultilineArrayBraceLayout
      - Layout/MultilineAssignmentLayout
      - Layout/MultilineHashBraceLayout
      - Layout/MultilineMethodCallBraceLayout
      - Layout/MultilineMethodCallIndentation
      - Layout/MultilineMethodDefinitionBraceLayout
      - Layout/MultilineOperationIndentation
      - Layout/TrailingBlankLines
      - Layout/TrailingWhitespace
      - Lint/BlockAlignment
      - Lint/EndAlignment
      - Lint/Void
      - Metrics/BlockLength
      - Metrics/BlockNesting
      - Metrics/LineLength
      - Naming/FileName
      - Style/FrozenStringLiteralComment
      - Style/IfUnlessModifier
      - Style/Next
      - Style/WhileUntilModifier
```

I would rather replace dashes with underscore in temporary files for a better fix and leave that linter enabled

**Comment by patientdev** [comment]
I'm still getting this error on `0.14`.

**Comment by patientdev** [comment]
Nevermind. I had to update the cop name in my own `.slim-lint.yml`
